test/fixtures/gcs-fixture/src/main/java/fixture/gcs/GoogleCloudStorageHttpHandler.java

```diff
@@ -394,4 +413,17 @@ public static int getContentRangeEnd(final String contentRange) {
             (bytes, limit) -> parse(PATTERN_CONTENT_RANGE_BYTES, bytes, (start, end) -> Integer.parseInt(end))
         );
     }
+
+    private static Long parseOptionalLongParameter(HttpExchange exchange, String parameterName) {
+        final Map<String, String> params = new HashMap<>();
+        RestUtils.decodeQueryString(exchange.getRequestURI().getQuery(), 0, params);
+        if (params.containsKey(parameterName)) {
+            try {
+                return Long.parseLong(params.get(parameterName));
+            } catch (NumberFormatException e) {
+                throw new MockGcsBlobStore.GcsRestException(RestStatus.BAD_REQUEST, "Invalid long parameter: " + parameterName);
```

Comment:
Could we assert this doesn't happen (i.e. fail the test if it does)?

Reply:
Fixed in fdf899242304caaaf22a0ae49ef4ecd02da32a93 and fdf899242304caaaf22a0ae49ef4ecd02da32a93